src/components/graph/graph.ts

```diff
@@ -30,11 +30,12 @@ import {
 } from './types'
 
 const logger = getLogger('d3-graph')
-if (window.localStorage.getItem('graphTransform') === undefined) {
+const temp1 = window.localStorage.getItem('graphTransform')
```

Comment:
Use a more descriptive variable name than temp.

Reply:
I was but then lint was complaining that the line was too long